app/js/actions/types.js

```diff
@@ -71,3 +76,17 @@ export const checkTypes = () => {
         .then(() => dispatch(refreshValidation()));
     };
 };
+
+
+export const checkImportableTypes = () => {
+    return (dispatch, getState) => {
+        const {
+            connection: { uri, secretKey },
+            superTypes: { importableTypes }
+        } = getState();
+        const url = `http://${uri}/api/types/importabletypes`;
```

Comment:
Since this is already under `/types/` why not make the endpoint `/types/importable`? (Not that this matters very much).